client/components/theme/index.jsx

```diff
@@ -40,6 +40,7 @@ export class Theme extends Component {
 			taxonomies: PropTypes.object,
 			update: PropTypes.object,
 			soft_launched: PropTypes.bool,
+			isCustomGeneratedTheme: PropTypes.bool,
```

Comment:
maybe this prop should be namedspaced harder...we have other kinds of _generated_ themes already, and might have others in the future